processor/k8sattributesprocessor/options.go

```diff
@@ -22,6 +22,7 @@ const (
 	filterOPExists       = "exists"
 	filterOPDoesNotExist = "does-not-exist"
 	metadataPodStartTime = "k8s.pod.start_time"
+	metadataPodIP        = "k8s.pod.ip"
```

Comment:
```suggestion
	metadataPodIP        = "k8s.pod.ip"
	metadataPodStartTime = "k8s.pod.start_time"
```
Nit: Swapping the lines keeps alphabetical order of constants.